src/views/partials/results/map_results.njk

```diff
@@ -0,0 +1,93 @@
+{% extends 'partials/modal/template.njk' %}
+
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{% set extraClass = ' map-modal'%}
+
+{% block beforeModalContent %}
+  <div class="govuk-grid-column-full">
+    <a href="javascript:;" {% if not searchMapResults.total %}disabled="true"{% endif %} onclick="openMapModal()" role="button" draggable="false" class="govuk-button govuk-button--map govuk-!-margin-bottom-4" data-module="govuk-button">
```

Comment:
If no results on the map we don't need to open the modal, the button is disabled but its functional.